packages/frontend-core/src/utils/rows.js

```diff
@@ -21,6 +21,8 @@ const isBetterSample = (newValue, oldValue) => {
   }
   if (
     newType === "object" &&
+    Array.isArray(newValue) &&
+    Array.isArray(oldValue) &&
```

Comment:
Do we need to add this? This was designed to work with both arrays and objects (like JSON columns). I guess if one if an array and one is an object then we don't want to consider this at all, but in that case I'd rather add a specific check up above this that if one is an array and one is not an array then we return false.

Reply:
I will split this up and I will change this `isArray` check to check nulls. This was the error, as we can have empty objects, and `Object.keys(null)` is not defined